src/map/script.cpp

```diff
@@ -12700,36 +12700,42 @@ BUILDIN_FUNC(gvgoff3)
 
 /*==========================================
  *	Shows an emoticon on top of the player/npc
- *	emotion emotion#, <target: 0 - NPC, 1 - PC>, <NPC/PC name>
+ *	emotion emotion#, <target: UNITTYPE_NPC (1) - NPC, UNITTYPE_PC (0) - PC>, <NPC/PC name>
```

Comment:
Remove the numbers please. Users should never use the values behind the constants.